src/control-plane/backend/lambda/api/service/project.ts

```diff
@@ -84,7 +84,7 @@ export class ProjectServ {
 
   public async createDashboard(req: any, res: any, next: any) {
     try {
-      const dashboardId = `clickstream-analysis-${uuidv4().replace(/-/g, '')}`;
+      const dashboardId = `clickstream-dashboard-${uuidv4().replace(/-/g, '')}`;
```

Comment:
Let's extract `-dashboard-` as constant for sharing it.

Reply:
done.